### DISCUSSION TITLE
Zeichnen/Redlining mobile

### DISCUSSION THREAD

**Discussion by marko-jer** [e-netze/webgis-community#54] [Q&A]
Sind die Objekte, die man beim Zeichnen/Redlining in der mobilen Version zur Verfügung hat, fix definiert? 
Oder kann man es individuell konfigurieren? 
![image](https://user-images.githubusercontent.com/1842500/231506572-32e45578-c626-40e8-8313-104e10f0965a.png)
Wenn ja, wie kann man da noch "Text" hinzufügen?

**Comment by jugstalt** [comment]
Kurze Antwort: Ja ( => Nein, nein) 

Es sind hier nur die Werkzeuge verfügbar, die auch mobil bedienbar sind. Texte sind dabei leider einmal nicht vorgesehen. Beschreibe vielleicht einmal den konrekten Anwendungsfall.

**Comment by marko-jer** [comment]
Konkreter Anwendungsfall wäre die simple Erfassung von Objekten mit dem "Text" als "Attribut" ohne Berechtigung und Konfiguration eines Edit-Themas. Funktioniert natürlich nur, wenn man den gleichen Wiederherstellungscode am Smartphone und PC verwendet.
Herunterladen als GeoJson sollte danach auch durchgeführt werden.
